flexmeasures/cli/tests/conftest.py

```diff
@@ -215,6 +215,7 @@ def storage_schedule_sensors(
 @pytest.fixture(scope="module")
 def add_asset_with_children(db, setup_roles_users):
     test_supplier_user = setup_roles_users["Test Supplier User"]
+    account_id = db.session.get(User, test_supplier_user).account_id
```

Comment:
A more direct approach would be 

`account_id = setup_accounts["Supplier"].id`

(and using that fixture in the signature, as well)